src/github.com/getlantern/flashlight/pac.go

```diff
@@ -41,16 +44,33 @@ func setUpPacTool() error {
 	return nil
 }
 
+func setProxyAddr(addr string) {
```

Comment:
Hmm, from http://www.aosabook.org/en/posa/high-performance-networking-in-chrome.html, DNS-prefetch may not work for proxied sites, especially when it uses [Built-in Asynchronous DNS](https://plus.google.com/+FrancoisBeaufort/posts/JXcvYw1yCkH) rather than system one.